web/app/charts/MapChart.js

```diff
@@ -22,11 +22,18 @@ bluewave.charts.MapChart = function(parent, config) {
             left: 82
         }
     };
-    var svg, mapArea; //d3 elements
+    var centeringConfig = {
+        newCenter: null
+    };
+    var svg, mapArea, mapCenter; //d3 elements
     var countyData, countryData; //raw json
     var counties, states, countries; //topojson
     var options = []; //aggregation options
-
+    var projection;
+    var zoomed = false;
+    var zoom = d3.zoom()
```

Comment:
This shouldn't be a class variable since it's not used outside of the constructor. Move this into the init() method.

Reply:
Will do